src/gwc/src/main/java/org/geoserver/gwc/GWC.java

```diff
@@ -762,8 +762,13 @@ public final ConveyorTile dispatch(
         }
 
         if (!tld.layerExists(layerName)) {
-            requestMistmatchTarget.append("not a tile layer");
-            return null;
+            // GEOS-9431-check in catalog also if not found TileLayerDispatcher
+            LayerInfo layerInfo = catalog.getLayerByName(layerName);
+            if (layerInfo != null) layerName = layerInfo.prefixedName();
```

Comment:
I'm assuming that the local workspace catalog wrapper is used here? In practice this means that:
- if not a virtual service request, the default workspace will be used
- if in the context of a virtual service request the local workspace will be used

Reply:
yes  that's correct